package-lock.json

```diff
@@ -2283,7 +2283,7 @@
         },
         "doctrine": {
           "version": "1.5.0",
-          "resolved": "http://registry.npmjs.org/doctrine/-/doctrine-1.5.0.tgz",
+          "resolved": "https://registry.npmjs.org/doctrine/-/doctrine-1.5.0.tgz",
```

Comment:
seems that file has been generated using a wrong npm version

Reply:
just rebuild it. It's like this with windows

Reply:
done, await travis to complete and will merge and deploy that soon